shared/constants/fs.js

```diff
@@ -242,3 +243,15 @@ export const makeDownloadKey = (path: Types.Path, localPath: string) =>
 
 export const downloadFilePathFromPath = (p: Types.Path): Promise<Types.LocalPath> =>
   downloadFilePath(Types.getPathName(p))
+export const downloadFilePathFromPathNoSearch = (p: Types.Path): string =>
+  downloadFilePathNoSearch(Types.getPathName(p))
+
+export const isImage = (name: string): boolean => {
+  const lower = name.toLowerCase()
+  for (let ext of ['.png', '.jpg', '.jpeg', '.mp4']) {
```

Comment:
should just use `['.png', '.jpg', '.jpeg', '.mp4'].some(ext => lower.endsWith(ext))`

Reply:
TIL...that's awesome!

Reply:
+1 TIL